src/resource/mock.test.ts

```diff
@@ -21,6 +21,9 @@
 
 import { describe, it, expect } from "@jest/globals";
 
+import * as crossFetch from "cross-fetch";
+const { Response } = crossFetch;
```

Comment:
As a nitpick and/or for my education: any reason this isn't just

```suggestion
import { Response } from "cross-fetch";
```

?

Reply:
Copied it from another file to be consistent 🤷‍♂️

Reply:
Ah, I see: https://github.com/inrupt/solid-client-js/commit/658d5901e7b1fb4f5ebb1fae1c889ce521a382fd

For posterity: that was because cross-fetch is a CommonJS module, and thus named imports will break when called from an ES module Node project. Not relevant here, but not problematic either - just good to know :)